ledcontroller/__init__.py

```diff
@@ -440,3 +453,26 @@ def batch_run(self, *commands):
                 args = command[1:]
                 cmd(*args)
         self.repeat_commands = original_retries
+
+
+
+def rgb_to_hue(red, green, blue):
```

Comment:
Furthermore, as this is not meant to be a generic library, I think the helper should take `RGB` tuple as an input, instead of separate red, green, blue.